### ISSUE TITLE
Using Translation Strings As Keys

### ISSUE THREAD

**Issue by moecasts** [GENL/matice#6]
Would you like to add `Using Translation Strings As Keys` feature like [https://laravel.com/docs/8.x/localization#using-translation-strings-as-keys](https://laravel.com/docs/8.x/localization#using-translation-strings-as-keys)

**Comment by GENL** [comment]
Hi! @MoeCasts. This feature will be supported soon. Still in development and will be available in the next release,

**Comment by GENL** [comment]
Take a look  at the PR https://github.com/GENL/matice/pull/5.
Once it will be finished, you'll be able to use `__()` function. It will try to find the string and if not found, returns the key,

**Comment by moecasts** [comment]
@GENL Thanks for your working, and I want to remind that the content of `/resource/lang/[lang].json` should be merged  into `matice_transitions.js` when run `matice:generate` command.

**Comment by tortuetorche** [comment]
>  want to remind that the content of `/resource/json/[lang].json` should be merged into `matice_transitions.js` when run `matice:generate` command.

Hi @MoeCasts,

Can you send here a reference of this behavior, please?
Because, I think Laravel only use this convention: [`/resources/lang/[lang].json`](https://laravel.com/docs/8.x/localization#using-translation-strings-as-keys).
So I tested to put a `/resources/json/[lang].json` file in my Laravel application, then I tried to translate a key from this file on the server side and it doesn't work...

Cheers,
Tortue Torche

**Comment by GENL** [comment]
> @GENL Thanks for your working, and I want to remind that the content of `/resource/json/[lang].json` should be merged into `matice_transitions.js` when run `matice:generate` command.

@MoeCasts , I tried it too. It seems Laravel doesn't work that way,

**Comment by MoeCasts** [comment]
@tortuetorche @GENL that's weird, it works for me.
that is my code.

```json
// ./resource/lang/zh_CN.json
{
    "Hello": "你好"
}
```

```json
// ./resource/lang/ja.json
{
    "Hello": "こんにちは"
}
```

```php
// ./config/app.php
<?php

return [
    // ...
    'locale' => 'zh_CN',
    
    //....
];
```

```php
// ./routes/web.php
<?php

use Illuminate\Support\Facades\Route;

Route::get('/', function () {
    echo __('Hello');
    return view('welcome');
});
```
![8D6188E8-9DFD-4047-AE43-1A3FF14D16AD](https://user-images.githubusercontent.com/37169906/98368992-2c360e00-2073-11eb-86d1-d6de79e7782d.png)

**Comment by moecasts** [comment]
@GENL @tortuetorche Sorry, it is my fault, i have written down the wrong path of the files, it should be `resource/lang/[lang].json`

**Comment by GENL** [comment]
Yes this works. But not in the `json` subdirectory. I think you made a mistake when you posted your question,

**Comment by tortuetorche** [comment]
I think you make a typo in [this comment](https://github.com/GENL/matice/issues/6#issuecomment-722720825)

I agree _/resource**s**/lang/zh_CN.json_ should be OK with the [master branch of matice](https://github.com/GENL/matice/tree/master) (see #2).
But not _/resource/**json**/zh_CN.json_ which doesn't work even with Laravel

**Comment by GENL** [comment]
Now available on release 1.1.2.
